.github/scripts/ansible/keycloak.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+cd $(dirname "${BASH_SOURCE[0]}")
+
+if [[ "$RUNNER_DEBUG" == "1" ]]; then
+  set -x
+fi
+
+REGION=$1
+KEYCLOAK_SRC=$2
+KEYCLOAK_M2=$3
```

Comment:
My IDE tells me that the environment variable `KEYCLOAK_M2` is not used. Please check.

Reply:
It's not, that's a leftover.